test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.FunctionalTests/ApplicationConsumingPrecompiledViews.cs

```diff
@@ -22,85 +16,26 @@ public ApplicationConsumingPrecompiledViews(ApplicationConsumingPrecompiledViews
 
         public ApplicationTestFixture Fixture { get; }
```

Comment:
Please either make test classes using the `ApplicationTestFixture` disposable (and dispose the `Fixture` there) or file an issue to do so.